.github/workflows/devcontainer-feature-release.yaml

```
@@ -0,0 +1,31 @@
+name: "Release dev container feature"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  call-workflow-test:
+    uses: ./.github/workflows/devcontainer-feature-test.yaml
+
+  
+  deploy:
+    needs: 
+      - call-workflow-test
+    if: ${{ github.ref == 'refs/heads/main' }}
```

```suggestion
    if: ${{ github.ref == 'refs/heads/main' && github.repository == 'radius-project/radius' }}
```

We special-case these for our repo so they don't run on forks and create a bunch of spam.